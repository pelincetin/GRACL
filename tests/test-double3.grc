void testdouble(double a, double b)
{
  printf(a + b);
  printf(a - b);
  printf(a * b);
  printf(a / b);
  printb(a == b);
  printb(a == a);
  printb(a != b);
  printb(a != a);
  printb(a > b);
  printb(a >= b);
  printb(a < b);
  printb(a <= b);
}

int main()
{
  double c;
  double d;

  c = 42.0;
  d = 3.14159;

  testdouble(c, d);

  testdouble(d, d);

  return 0;
}
