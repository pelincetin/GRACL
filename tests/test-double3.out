45.141590
38.858410
131.946780
13.369027
False
True
True
False
True
True
False
False
6.283180
0.000000
9.869588
1.000000
True
True
False
False
False
True
False
True
